mkosi/config.py

```diff
@@ -488,6 +477,16 @@ def config_parse_root_password(value: Optional[str], old: Optional[tuple[str, bo
     return (value, hashed)
 
 
+def config_parse_tools_tree(value: Optional[str], old: Union[Path, None, Literal["default"]]) -> Union[Path, None, Literal["default"]]:
```

Comment:
`Union[Path, None, Literal["default"]]` is a bit of a mouthful, how about defining a type alias `OptionalOrDefaultPath` (or similar)?

Reply:
New revision drops the `Union` stuff altogether